NthKey/Model/Configuration.storekit

```diff
@@ -0,0 +1,79 @@
+{
+  "products" : [
+
+  ],
+  "settings" : {
+    "_timeRate" : 0
+  },
+  "subscriptionGroups" : [
+    {
+      "id" : "5BC28C3C",
+      "localizations" : [
+
+      ],
+      "name" : "subscription_group",
+      "subscriptions" : [
+        {
+          "adHocOffers" : [
+            {
+              "internalID" : "F0D0AE94",
+              "offerID" : "",
+              "paymentMode" : "free",
+              "referenceName" : "",
+              "subscriptionPeriod" : "P3D"
+            }
+          ],
+          "displayPrice" : "0.99",
+          "familyShareable" : false,
+          "groupNumber" : 1,
+          "internalID" : "2DB11FFB",
+          "introductoryOffer" : null,
+          "localizations" : [
+            {
+              "description" : "",
+              "displayName" : "",
+              "locale" : "en_US"
+            }
+          ],
+          "productID" : "com.nthkey.monthly",
+          "recurringSubscriptionPeriod" : "P1M",
+          "referenceName" : "Monthly",
+          "subscriptionGroupID" : "5BC28C3C",
+          "type" : "RecurringSubscription"
+        },
+        {
+          "adHocOffers" : [
+            {
+              "internalID" : "0A0F6456",
+              "offerID" : "",
+              "paymentMode" : "free",
+              "referenceName" : "Test week",
+              "subscriptionPeriod" : "P1W"
+            }
+          ],
+          "displayPrice" : "4.99",
+          "familyShareable" : true,
+          "groupNumber" : 1,
+          "internalID" : "F3FCF59A",
+          "introductoryOffer" : null,
+          "localizations" : [
+            {
+              "description" : "",
+              "displayName" : "",
+              "locale" : "en_US"
+            }
+          ],
+          "productID" : "com.nthkey.annual",
```

Comment:
Looks like you forgot remove this in 9b52487.

Reply:
This is test configuration which didn't affect to real iAP